doc/SPIRE101.md

```diff
@@ -0,0 +1,138 @@
+    
+        make build
+
+7.  Try running help for register sub command. The **spire-server** and **spire-agent** executables have `-—help`  option that give details of respective cli options. 
+	
+	    ./cmd/spire-server/spire-server register --help
+
+8.  View the server configuration file.
+    	 
+    	    cat conf/server/server.conf
+    
+    The default server configurations are shown below. A detailed description of each of the server configuration option is [here](/README.md#spire-server-configuration)
+    
+    ```
+    BaseSVIDTTL = 999999
+    ServerSVIDTTL = 999999
+    BindAddress = "127.0.0.1"
+    BindPort = "8081"
+    BindHTTPPort = "8080"
+    LogLevel = "INFO"
+    PluginDir = "conf/server/plugin"
+    TrustDomain = "example.org"
+    Umask = ""
+    ```
+
+9.  Start the spire-server as a background process by running the following command.
+
+        ./cmd/spire-server/spire-server run &
+
+10. Generate a one time join token via **spire-server token generate** sub commmand. Use the **-spiffeID** option to associate the join token with **spiffe://example.org/host** Spiffe ID. Save the genreated join token in your copy buffer.
+	
+	    ./cmd/spire-server/spire-server token generate -spiffeID spiffe://example.org/host
+	    
+	 The join token will be used for node attestation and the associated spiffeID will be used to generate the SVID of the attested node. 
+	 
+	 The default ttl of the join token is 600 seconds. We can overwrite the default value through **-ttl** option.
+
+11. View the configuration file of the agent
+    	
+        cat conf/agent/agent.conf
+    
+    The default agent configurations are shown below. A detailed description of each of the agent configuration option is [here](/README.md#spire-agent-configuration)
+    ```
+    BindAddress = "127.0.0.1"
+    BindPort = "8088"
+    DataDir = "."
+    LogLevel = "INFO"
+    PluginDir = "conf/agent/plugin"
+    ServerAddress = "127.0.0.1"
+    ServerPort = "8081"
+    SocketPath ="/tmp/agent.sock"
+    TrustBundlePath = "conf/agent/carootcert.pem"
+    TrustDomain = "example.org"
+    Umask = ""
+    ```
+
+12. Start the spire-agent as a background process by replacing **_<generated-join-token>_** with the generated join token in step #10 and running the following command.
+
+        ./cmd/spire-agent/spire-agent run -joinToken <generated-join-token> &
+
+13. The next step is to register a Spiffe ID with a set of selectors. For the example we will use unix kernel selectors that will be mapped to a target SPIFFE ID.
+   	
+        ./cmd/spire-server/spire-server register -parentID spiffe://example.org/host -spiffeID spiffe://example.org/host/workload -selector unix:uid:1000
+    At this point, the registration API has been called and the target workload has been registered with the SPIRE Server. We can now call the workload API using a command line program to request a new SVID from the SPIRE server.
+
```

Comment:
Should request the SVID from the SPIRE agent not the server. agent implements the Workload API